src/app/modules/submission/receiver/receiver.utils.ts

```diff
@@ -79,7 +79,6 @@ export const addAttachmentToResponses = (
     responses.forEach((response) => {
       if (isAttachmentResponseFromMap(attachmentMap, response)) {
         const file = attachmentMap[response._id]
-        response.answer = file.filename
         response.filename = file.filename
```

Comment:
qn: why change the key here to filename; is this a breaking change?

Reply:
Oh I'm just removing the force setting of `response.answer` to `file.filename` so I can pass the quarantine file key in the answer without it being overwritten. So a request to the virus scan enabled endpoint can still succeed while triggering the lambda in parallel.

If I were to leave the setting of `response.answer` to `file.filename` here, I can set the filename to the quarantine file key and trigger the lambda scan, but then the validator down the line will complain that the file doesn't have a valid file extension.

Reply:
I don't think this is a breaking change because based on my observation of submissions, the FE should already be setting the `response.answer` as the filename.

Reply:
Okay on second thought, playwright tests might be failing because of this, meaning there'll be a regression. I've restored this in [eec5388](https://github.com/opengovsg/FormSG/pull/6734/commits/eec53881d6d5e11d27a2963b73d9b23705be06c9). We can bare with the invalid file extension error since the virus scanning flag should be unset or off when this goes out.